src/main/java/seedu/duke/parsers/OptionParser.java

```diff
@@ -8,7 +8,8 @@
 
 public class OptionParser extends Parser {
 
-    private static final String OPTION_FORMAT = "(\\s*(?<configurationGroupWord>[A-Z_]+)(\\s*=\\s*(?<newValue>.*))?)?";
+    private static final String OPTION_FORMAT = "(\\s*(?<configurationGroupWord>[A-Z_]+)"
```

Comment:
this regex does not achieve the option CONFIG_NAME=NEW_VALUE because of the \s* before newValue, but this change should be resolved in the other PRs

Reply:
This problem will be fixed by latter pr #121 by heekit.